ddr/ddr/Models/GameEntities/Oncomer.swift

```diff
@@ -79,12 +79,20 @@ class Oncomer: SKSpriteNode, Spawnable {
     self.collisionEffects = self.collisionEffects + [ KillOncomerEffect(oncomer: self) ]
     self.passEffects = self.passEffects + [ HideEffect(nodeToHide: self) ]
     self.goneEffects = self.goneEffects + [ KillOncomerEffect(oncomer: self) ]
+    animate()
   }
   
   required init?(coder aDecoder: NSCoder) {
     fatalError("init(coder:) has not been implemented")
   }
   
+  func animate(){
```

Comment:
I already wrote this function - did you pull? I believe mine is more flexible...